ctms/models.py

```diff
@@ -186,3 +187,23 @@ class MozillaFoundationContact(Base):
     mofo_relevant = Column(Boolean)
 
     email = relationship("Email", back_populates="mofo", uselist=False)
+
+
+class PendingAcousticRecord(Base):
+    __tablename__ = "pending_acoustic"
+
+    id = Column(Integer, Sequence("pending_id_sequence"), primary_key=True)
```

Comment:
I think the ``Sequence`` can be dropped. Other models like ``Newsletter`` have an auto-created ``id`` sequence that seems to do the job. More details on the migration.